File: aptcompiler/src/main/java/com/eugene/aptcompiler/utils/TypeUtil.java
package com.eugene.aptcompiler.utils;

import com.squareup.javapoet.ClassName;

public class TypeUtil {
    public static final ClassName ANDROID_VIEW = ClassName.get("android.view", "View");
    public static final ClassName ANDROID_ON_CLICK_LISTENER = ClassName.get("android.view", "View", "OnClickListener");
    public static final ClassName BINDER = ClassName.get("com.example.aop.apt", "Binder");
    public static final ClassName PROVIDER = ClassName.get("com.example.aop.apt.provider", "Provider");

    public static final ClassName RESOLVER = ClassName.get("com.eugene.aop.router", "Resolver");
    public static final ClassName ACTIVITY = ClassName.get("android.app", "Activity");
    public static final ClassName INTENT = ClassName.get("android.content", "Intent");
    public static final ClassName BUNDLE = ClassName.get("android.os", "Bundle");
}

File: aptcompiler/src/main/java/com/eugene/aptcompiler/kutils/TypeUtil.kt
package com.eugene.aptcompiler.kutils

import com.squareup.kotlinpoet.ClassName


val ANDROID_VIEW = ClassName.bestGuess("android.view.View")
val ANDROID_ON_CLICK_LISTENER = ClassName.bestGuess("android.view.View.OnClickListener")
val BINDER = ClassName.bestGuess("com.example.aop.apt.Binder")
val PROVIDER = ClassName.bestGuess("com.example.aop.apt.provider.Provider")

val RESOLVER = ClassName.bestGuess("com.eugene.aop.router.Resolver")
val ACTIVITY = ClassName.bestGuess("android.app.Activity")
val INTENT = ClassName.bestGuess("android.content.Intent")
val BUNDLE = ClassName.bestGuess("android.os.Bundle")
